dataplane/internal/engine/engine.go

```diff
@@ -430,9 +442,24 @@ func (e *Engine) addNextHopGroupIDList(ctx context.Context, id uint64, nhg *dpb.
 			Actions: actions,
 		}},
 	}
+
+	if mode == dpb.GroupUpdateMode_GROUP_UPDATE_MODE_APPEND || mode == dpb.GroupUpdateMode_GROUP_UPDATE_MODE_REPLACE {
```

Comment:
handle all enum values with a default case. If new one is added and this is not updated then UNSPECIFIED's behaviour will be used instead of throwing an error.

Reply:
done